22.md

```diff
@@ -0,0 +1,56 @@
+NIP-22
+======
+
+Reactions v2
+------------
+
+`draft` `optional`
+
+This NIP should be used in place of [NIP-25](25.md).
+
+A reaction is a way to like/upvote or dislike/downvote an event or any entity external to nostr,
+such as an URL.
+
+A like is a `kind:31143` event. A dislike is a `kind:31144` one.
+
+The `d` tag is the address of the event being reacted to, if applicable, or the event's id or
+any other identifier from [NIP-73](73.md)'s `s` tag possible values.
+
+The `.content` field is one of: emoji, custom emoji (e.g.: ":emoji_shortcode_example:") or empty string.
+If a custom emoji, it must include a [NIP-30](30.md) `emoji` tag.
+
+One `p` tag must be included when reacting to an event, with the reacted to event's pubkey. It should
```

Comment:
why keep a reference to the author and not the post? since the port contains the `"pubkey"` as well.

Reply:
im not sure about nip-65, but for notification, nostr is not friendly with notifications at all as far as i know.

Reply:
It is not a notification in that sense in this context.

Reply:
oh, so what kind of notifs you mean?

Reply:
It is almost just convention than the notification.